### PR TITLE
feat(wallet): Added interface to send debit wallet requests

### PR DESCRIPTION
**Pull request by cheezytony**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Services/CashService.php [hunk 1]
```diff
@@ -204,6 +204,25 @@ public function withdrawFromWallet(string $walletId, string $amount, string $des
...
+    /**
+     * @description Debit a wallet
+     * @param string $walletId
+     * @param string $amount
+     * @param string $description
+     * @return array|mixed
+     * @throws CPCashException
+     * @throws InternalServerException|NotFoundException
+     */
+    public function debitWallet(string $walletId, string $amount, string $description)
```
Comment (dammy001) [inline src/Services/CashService.php:216]: any difference with this method `public function withdrawFromWallet(string $walletId, string $amount, string $description)`?

**Review by dammy001 — CHANGES_REQUESTED** [review]

File: src/Services/CashService.php (hunk 1 above)
Comment (cheezytony) [reply src/Services/CashService.php:216]: This one updates the ledger balance immediately

**Comment by dammy001** [comment]
LGTM

File: src/Contracts/CPCash.php [hunk 2]
```diff
@@ -83,6 +83,17 @@ public function walletTopUpWithReference(
      */
     public function withdrawFromWallet(string $walletId, string $amount, string $description);
 
+    /**
+     * @description Withdraw wallet
```
Comment (dammy001) [inline src/Contracts/CPCash.php:87]:
```suggestion
     * @description Debit wallet
```

**Pull request merged**